projects/ui/src/lib/components/po-menu/po-menu.component.spec.ts

```diff
@@ -644,19 +644,21 @@ describe('PoMenuComponent:', () => {
     });
 
     it('should show `po-menu-footer` if `hasFooter` is `true`', () => {
```

Comment:
Verificar o teste.

A descrição do it não esta condizente com o que ele realmente faz.

```
it('should show `po-menu-footer` if `collapsed` is `true` and `menus` are valid', () => {
      component.collapsed = true;
      component.menus = [{ label: '1', icon: 'po-icon-user', shortLabel: '123', action: () => {} }];

      fixture.detectChanges();
      const footer = fixture.debugElement.nativeElement.querySelector('.po-menu-footer');

      expect(footer).toBeTruthy();
});
```